WebScraping/docker-lambda-function/lambda_function.py

```diff
@@ -24,61 +73,74 @@ def handler(event=None, context=None):
     options.add_argument("--remote-debugging-port=9222")
 
     driver = webdriver.Chrome(options=options, service=service)
-    driver.implicitly_wait(5)
-    driver.get("https://central.carleton.ca/prod/bwysched.p_select_term?wsea_code=EXT")
-
-    hrefs = set()
+    driver.get(PAGE_SOURCE)
+    
+    return driver
 
+def scrape_terms(driver):
```

Comment:
Nit: I suggest adding type hints where you are missing them (e.g. in this case specify the type of the parameter and that the return type is List[str]). You will probably need to import "typing".

Reply:
Yes, this is planned to be completed soon